src/store/modules/user/UserState.ts

```diff
@@ -4,4 +4,5 @@ export default interface UserState {
     currentEComStore: object;
     instanceUrl: string;
     shopifyConfig: object;
+    statusDesc: any;
```

Comment:
I think it should part of util

Reply:
I have created a new module util and stored status related information in it